### PR TITLE
[chore]: add enum keys for config store

### PR DESCRIPTION
**Pull request by 12ya**
[chore]: add enum keys for config store

- [x] Tested the feature and it's working on a current and clean install.
- [x] Tested the main App features and they are still working on a current and clean install. (Login, Install, Play, Uninstall, Move games, etc.)
- [ ] Created / Updated Tests (If necessary)
- [ ] Created / Updated documentation (If necessary)

### CODE REVIEW

**Comment by flavioislima** [comment]
**CLA Assistant Lite bot:** <br/>Thank you for your submission, we really appreciate it. Like many open-source projects, we ask that you sign our [Contributor License Agreement](https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/blob/main/doc/cla.md) before we can accept your contribution. You can sign the CLA by just posting a Pull Request Comment same as the below format.<br/>
   - - -
   I have read the CLA Document and I hereby sign the CLA
   - - -
   <sub>You can retrigger this bot by commenting **recheck** in this Pull Request</sub>

File: pnpm-lock.yaml
Comment (arielj) [inline pnpm-lock.yaml:1]: I don't think this changes belong to this PR

**Comment by arielj** [comment]
what's the reason for all these `enum` PRs? what si this solving? or are the changes just some "good practice" pattern? is there something these PRs are fixing?

**Comment by 12ya** [comment]
> what's the reason for all these `enum` PRs? what si this solving? or are the changes just some "good practice" pattern? is there something these PRs are fixing?

main reason is to add safety by replacing strings; if there is better solution for this, please guide me and I'll try to implement it

File: pnpm-lock.yaml
Comment (12ya) [reply pnpm-lock.yaml:1]: removed

**Comment by arielj** [comment]
Closing, this PR has changes from another PR (changing en-dash to hypen) and a change in the lock file, so it's not really changing what it says in the title

**Pull request closed without merging**